docs/pages/enterprise/workflow/index.mdx

```diff
@@ -1,281 +1,48 @@
 ---
-title: Access Requests for Infrastructure Access
-description: Teleport allows users to request new roles with elevated privileges from the CLI or UI. Requests can be escalated via ChatOps or anywhere else via our flexible Authorization Workflow API.
-h1: Teleport Access Requests
+title: Just-in-time Access Requests
+description: Teleport allows users to request new access capabilities from the CLI or UI. Requests can be escalated via ChatOps or anywhere else via our flexible Authorization Workflow API.
+h1: Teleport Just-in-time Access Requests
 ---
 
-With Teleport, users can request additional roles via a third-party
-communication service. The Access Request API makes it easy to dynamically
-approve or deny these requests.
+Teleport Just-in-time Access Requests allow any developer to request access to
+a resource or role depending on need. The request can then be approved or
+denied based on a configurable number of approvers.
 
-<ScopedBlock
-  scope={["oss"]}
->
+<ScopedBlock scope={["oss"]}>
 
-  This guide requires Teleport Cloud or Teleport Enterprise.
+Just-in-time Access Requests are a feature of Teleport Enterprise.
+Open-source Teleport users can get a preview of how Access Requests work by
+requesting a role via the Teleport CLI. Full access request functionality,
+including Resource Access Requests and an intuitive and searchable UI are
+available in Teleport Enterprise.
 
-  View this guide as the user of another Teleport edition:
+View this guide as the user of another Teleport edition:
 
-  <TileSet>
-  <Tile icon="cloud" title="Teleport Cloud" href="./index.mdx/?scope=cloud">
-  </Tile>
-  <Tile icon="building" title="Teleport Enterprise" href="./index.mdx/?scope=enterprise">
-  </Tile>
-  </TileSet>
+<TileSet>
+<Tile icon="cloud" title="Teleport Cloud" href="./index.mdx/?scope=cloud">
+</Tile>
+<Tile icon="building" title="Teleport Enterprise" href="./index.mdx/?scope=enterprise">
+</Tile>
+</TileSet>
 
 </ScopedBlock>
 
 <ScopedBlock scope={["cloud", "enterprise"]}>
 
-- [Integrating Teleport with Slack](ssh-approval-slack.mdx)
-- [Integrating Teleport with Mattermost](ssh-approval-mattermost.mdx)
-- [Integrating Teleport with Jira Cloud](ssh-approval-jira-cloud.mdx)
-- [Integrating Teleport with Jira Server](ssh-approval-jira-server.mdx)
-- [Integrating Teleport with PagerDuty](ssh-approval-pagerduty.mdx)
+## Resource Access Requests
 
-## Access Requests Setup
+With Resource Access Requests, an engineer can request access to one or more
+individual resources they need, when they need it.<br/>
+[Get started with Resource Access Requests](./resource-requests.mdx).
```

Comment:
Would it be worth mentioning "least privilege" here somewhere? Might help with SEO and would also highlight that this is a more secure option.

Reply:
@zmb3 What do you think of
> Resource Access Requests follow the principle of least privilege. On teams leveraging Teleport, engineers can easily get access to only the individual resources they need, when they need it.

The blurb for Role Access Requests could become
> Role Access Requests balance security and flexibility. Engineers can request temporary credentials with elevated roles in order to perform critical system-wide tasks.

Reply:
I updated the copy here to this ^